pkg/config/data.go

```diff
@@ -539,34 +447,34 @@ func (f *File) Read(path string, in io.Reader, out io.Writer, errLog fsterr.LogI
 		// version embedded into the CLI binary.
 
 		text.Break(out)
-		label := fmt.Sprintf("Your configuration file (%s) is invalid. Replace it with a valid version? (any existing email/token data will be lost) [y/N] ", path)
-		cont, err := text.Input(out, label, in)
+
+		label := fmt.Sprintf("Your configuration file (%s) is invalid. %s", path, replacement)
+		cont, err := text.AskYesNo(out, label, in)
 		if err != nil {
-			return fmt.Errorf("error reading input %w", err)
+			return fmt.Errorf("error reading input: %w", err)
 		}
-		contl := strings.ToLower(cont)
-		if contl == "y" || contl == "yes" {
-			data = f.static
-
-			Mutex.Lock()
-			err = toml.Unmarshal(data, f)
-			Mutex.Unlock()
-
-			if err != nil {
-				errLog.Add(err)
-				return invalidConfigErr(err)
-			}
-		} else {
+		if !cont {
 			err := fsterr.RemediationError{
 				Inner:       fmt.Errorf("%v: %v", ErrInvalidConfig, unmarshalErr),
 				Remediation: RemediationManualFix,
 			}
 			errLog.Add(err)
 			return err
 		}
+
+		data = f.static
+
+		Mutex.Lock()
+		err = toml.Unmarshal(data, f)
+		Mutex.Unlock()
```

Comment:
Do we still need the mutexes?

Reply:
Removed, with the exception of the Write method (as documented in its comments).